doc/python/sphinx/grpc_asyncio.rst

```diff
@@ -24,27 +24,20 @@ gRPC Async API objects may only be used on the thread on which they were
 created. AsyncIO doesn't provide thread safety for most of its APIs.
 
 
-Module Contents
----------------
-
 Enable AsyncIO in gRPC
-^^^^^^^^^^^^^^^^^^^^^^
+----------------------
 
-.. function:: init_grpc_aio
+Enable AsyncIO in gRPC Python is automatic when instantiating gRPC AsyncIO
```

Comment:
I don't think this paragraph is actually necessary. I doubt people will assume that setup is necessary unless instructed otherwise.

Reply:
Removed the first paragraph, and changed the title to `Blocking Code in AsyncIO`.